includes/class-wc-calypso-bridge-ecommerce-admin-menu.php

```diff
@@ -59,8 +58,12 @@ public static function menu_order( $menu_order ) {
 		$extensions_index          = array_search( 'woocommerce', $menu_order, true );
 		$woocommerce_sep_index     = array_search( 'separator-woocommerce', $menu_order, true );
 		$woocommerce_sep_top_index = array_search( 'wc-calypso-bridge-separator-top', $menu_order, true );
+
+		// Get the index of our managed pages for integrations.
 		$payments_connect_index    = array_search( 'wc-admin&path=/payments/connect', $menu_order, true );
```

Comment:
Do we care about the indexes in general in this function? Isn’t `in_array` enough? (Unless `array_search` is more performant)